planner/core/planbuilder.go

```diff
@@ -2871,6 +2871,15 @@ func (b *PlanBuilder) buildShow(ctx context.Context, show *ast.ShowStmt) (Plan,
 	}.Init(b.ctx)
 	isView := false
 	isSequence := false
+	switch show.Tp {
+	case ast.ShowColumns:
```

Comment:
Why not merge this switch-case to the switch-case below?

Reply:
Done